cmake/templates/oneDPLConfig.cmake.in

```diff
@@ -52,6 +52,11 @@ if (EXISTS "${_onedpl_headers}")
         add_library(oneDPL INTERFACE IMPORTED)
         set_target_properties(oneDPL PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${_onedpl_headers}")
 
+        if (WIN32)
+            # oneDPL is incompatible with min/max macros defined in windows.h, NOMINMAX must be defined to avoid them
+            set_target_properties(oneDPL PROPERTIES INTERFACE_COMPILE_OPTIONS -DNOMINMAX)
```

Comment:
Minor suggestion: Should we add some message here for the user (e.g. "Defining NOMINMAX to avoid collisions with Windows macros.")?

Reply:
Thats a good idea, I can add it.